.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
<img width="1279" alt="image" src="https://github.com/rootstrap/rails_api_base/assets/6373536/fff37443-cbfe-4da3-a4c1-1299e97ceaf0">

we already have dependabot, maybe it's just a permissions thing?

Reply:
I think those are `Dependabot security updates`. What I want to enable is `Dependabot version updates`, which creates new PRs so that developers review and merge: https://github.com/rootstrap/rails_api_base/network/updates

```
Dependabot security updates are automated pull requests that help you update dependencies with known vulnerabilities.
Dependabot version updates are automated pull requests that keep your dependencies updated, even when they don’t have any vulnerabilities. To check the status of version updates, navigate to the Insights tab of your repository, then Dependency Graph, and Dependabot.
```

Reply:
ahhh cool